importexport/musicxml/exportxml.cpp

```diff
@@ -4598,50 +4577,109 @@ static bool commonAnnotations(ExportMusicXml* exp, const Element* e, int sstaff)
       }
 
 //---------------------------------------------------------
-//  annotations
+//  anotations
 //---------------------------------------------------------
 
-/*
- * Write annotations that are attached to chords or rests
- */
-
-// In MuseScore, Element::FRET_DIAGRAM and Element::HARMONY are separate annotations,
-// in MusicXML they are combined in the harmony element. This means they have to be matched.
-// TODO: replace/repair current algorithm (which can only handle one FRET_DIAGRAM and one HARMONY)
+// Only handle common annotations, others are handled elsewhere
 
 static void annotations(ExportMusicXml* exp, int strack, int etrack, int track, int sstaff, Segment* seg)
       {
-      if (seg->segmentType() == SegmentType::ChordRest) {
 
-            const FretDiagram* fd = findFretDiagram(strack, etrack, track, seg);
-            // if (fd) qDebug("annotations seg %p found fretboard diagram %p", seg, fd);
+      for (const Element* e : seg->annotations()) {
 
-            for (const Element* e : seg->annotations()) {
+            int wtrack = -1; // track to write annotation
```

Comment:
Despite my best efforts, I still don't understand what are those `wtrack` checks are used for. Since I'm not too sure, I did'nt add such a check in the following `harmonies` method. Fill free to tell me where I'm wrong.

Reply:
I feel like this might be [related to this issue](https://musescore.org/en/node/303450)?

Reply:
Wtrack is the track where something is to be written. Required when MuseScore forces things in a staff's first track (e.g. staff text), when no note start at that time. This seems to no longer apply to harmonies.